examples/calculator/pom.xml

```diff
@@ -6,7 +6,7 @@
     <description>A simple calculator application</description>
     <groupId>com.spotify</groupId>
     <artifactId>calculator-service</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
This one should probably not be versioned with the main artifacts. The example is not supposed to be published, so it should stay on 0.0.1-SNAPSHOT